src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php

```diff
@@ -98,7 +98,8 @@ public static function dataFieldsProvider()
      */
     public function testGetFields($request)
     {
-        $helper = new ListRestHelper($request, $this->em);
+        $this->requestStack->method('getCurrentRequest')->willReturn($request);
```

Comment:
Would I claim too much if I ask you to also get rid of the PHPUnit Mock, and replace it with the prophecy one? 🙈